aws_s3_access_grants_boto3_plugin/cache/account_id_resolver_cache.py

```diff
@@ -2,7 +2,7 @@
 from aws_s3_access_grants_boto3_plugin.exceptions import IllegalArgumentException
 
 DEFAULT_ACCOUNT_ID_CACHE_SIZE = 1000
-DEFAULT_TTL = 600
+DEFAULT_TTL = 60 * 60
```

Comment:
Question - Have we discussed with AWS SDK team regarding potential risks with regarding to this cache TTL increase?